src/Reflection/Adapter/ReflectionAttribute.php

```diff
@@ -54,4 +58,13 @@ public function __toString(): string
     {
         return $this->betterReflectionAttribute->__toString();
     }
+
+    public function __get(string $name): mixed
```

Comment:
Did we add this because upstream declares the variable? Can't we set `name` in the constructor? :thinking:

Reply:
Yes, it's change in PHP 8.4: https://github.com/php/php-src/blob/PHP-8.4/UPGRADING#L409

It's not possible to change it in constructor: https://3v4l.org/lgklF#v8.4.1
I've used the some pattern as used in other adapters.